job_executor/worker/build_dataset_worker.py

```diff
@@ -25,9 +27,13 @@ def _clean_working_dir(dataset_name: str):
         WORKING_DIR / f"{dataset_name}_enriched.csv",
         WORKING_DIR / f"{dataset_name}_pseudonymized.csv",
         WORKING_DIR / f"{dataset_name}.db",
+        WORKING_DIR / dataset_name,
     ]
     for file_path in generated_files:
-        local_storage.delete_working_dir_file(file_path)
+        if file_path.is_dir():
```

Comment:
Did you intend to use the new func from localstorage here 👀 ? delete_working_dir_dir

Reply:
Yes, you are correct. Will fixing it for build_metadata too.